build.xml

```diff
@@ -1,256 +0,0 @@
-<fail message="JTS Topology Suite JAR file ${jts.jar} not found.  Download it from http://tsusiatsoftware.net" unless="jts.jar.isAvailable"/>
-
-<!-- 'Java PROJ.4 library' JAR file required for building some Java classes -->
-<property name="proj4.jar" value="javaproj-1.0.9.jar"/>
-<available file="${proj4.jar}" property="proj4.jar.isAvailable"/>
-<fail message="Java PROJ.4 library JAR file ${proj4.jar} not found.  Download it from http://www.jhlabs.com" unless="proj4.jar.isAvailable"/>
-
-<!-- servlet-api.jar JAR file required for building servlet -->
-<property name="servlet-api.jar" value="servlet-api.jar"/>
-<available file="${servlet-api.jar}" property="servlet-api.jar.isAvailable"/>
-<fail message="Servlet API JAR file ${servlet-api.jar} not found.  Download it as part of Tomcat from http://tomcat.apache.org" unless="servlet-api.jar.isAvailable"/>
-
-<!-- Oracle Spatial JAR files required for building some Java classes -->
-<property name="oracle.jars" value="sdoapi.jar:ojdbc14.jar"/>
-<condition property="oracle.jars.areAvailable">
-<and>
-<available file="sdoapi.jar"/>
-<available file="ojdbc14.jar"/>
-</and>
-</condition>
-<fail message="Oracle Spatial JAR files ${oracle.jars} not found." unless="oracle.jars.areAvailable"/>
-
-<tstamp>
-<format property="today" pattern="d-MMMM-yyyy" locale="en"/>
-</tstamp>
-</target>
-
-<target name="compile" depends="init" description="Compile java source code">
-<!-- Create version and date stamp files for JAR file and documentation -->
-<echo file="userdoc/version.txt">${version}</echo>
-<echo file="userdoc/timestamp.txt">${today}</echo>
-<echo file="src/org/mapyrus/util/version.txt">${version}</echo>
-<echo file="src/org/mapyrus/util/timestamp.txt">${today}</echo>
-
-<mkdir dir="build/classes"/>
-<javac srcdir="src" listfiles="yes" classpath="${jts.jar}:${proj4.jar}:${oracle.jars}:${servlet-api.jar}" destdir="build/classes"/>
-
-<!-- Copy resource, font and data files into built directory -->
-<copy file="src/org/mapyrus/util/version.txt" todir="build/classes/org/mapyrus/util"/>
-<copy file="src/org/mapyrus/util/timestamp.txt" todir="build/classes/org/mapyrus/util"/>
-<copy file="src/org/mapyrus/Messages.properties" todir="build/classes/org/mapyrus"/>
-<copy file="src/org/mapyrus/gui/onlinehelp.html" todir="build/classes/org/mapyrus/gui"/>
-<copy file="src/org/mapyrus/gui/commands1.txt" todir="build/classes/org/mapyrus/gui"/>
-<copy file="src/org/mapyrus/gui/commands2.txt" todir="build/classes/org/mapyrus/gui"/>
-<copy file="src/org/mapyrus/gui/commands3.txt" todir="build/classes/org/mapyrus/gui"/>
-<copy todir="build/classes/org/mapyrus/dataset">
-<fileset dir="src/org/mapyrus/dataset" includes="*.txt,*.jpg,*.tfw"/>
-</copy>
-<copy file="userdoc/coastline.shp" todir="build/classes/org/mapyrus/dataset"/>
-<copy file="userdoc/australia.png" todir="build/classes/org/mapyrus/dataset"/>
-<copy file="userdoc/australia.tfw" todir="build/classes/org/mapyrus/dataset"/>

```

Comment:
Those three files are currently not included in the build. Do we need them?

Reply:
These files are not needed.

These files were included in `mapyrus.jar` so that the user can run one of the examples in the manual (4.15, "Displaying Datasets and Images Stored In JAR Files"). That example uses SHP and PNG files packed in `mapyrus.jar`.

I can update this section of the Mapyrus manual to explain that the user has to pack their own data files into their own JAR file.

Reply:
Alright, so I don't have to adapt the build in this respect.

Please let me know if other modifications are needed. Yesterday, I've checked by diffing the file listing in the .jar/.war file against the currently released 1.802 and found no problematic differences.

Reply:
After your changes, all of the examples in the Mapyrus PDF manual are still created correctly.

I will merge your Pull Request.